src/style.css

```diff
@@ -1,3 +1,87 @@
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-left: 50px;
+}
+
+.logo_link {
+  height: 60px;
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  margin-right: 50px;
+  padding: 0;
+}
+
+.u-list {
+  margin: 0;
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+}
+
+.nav_list_item {
+  list-style-type: none;
+  display: flex;
+}
+
+.nav_link {
+  margin: 0 10px;
+  line-height: 60px;
+  height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  font-size: 12px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.nav_link:hover {
+  color: var(--blue);
+}
+
+a.is-active {
+  color: var(--blue);
+  margin-left: 0;
+  position: relative;
+}
+
+a.is-active::after {
+  position: absolute;
+  content: url(./images/Rectangle.png);
```

Comment:
Don't add underline as a photo. What if you need to add it for another link?

Reply:
I added some changes due to Your recommendations